### ISSUE TITLE
Change "Retry" button UI in inline error notice to visually appear like an inline link

### ISSUE THREAD

**Issue by felixarntz** [google/site-kit-wp#6055]
Follow-up to #5494: While that issue correctly implements the "Retry" button in inline notices (e.g. in module setup flows), having a primary CTA button in those error notices looks a bit odd (see https://github.com/google/site-kit-wp/issues/5494#issuecomment-1289347228).

They should become a regular inline link instead.

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance criteria

* 

## Implementation Brief

* 

### Test Coverage

* 

## QA Brief

* 

## Changelog entry

*

**Comment by techanvil** [comment]
Hi @nfmohit, thanks for the IB.

I've got an alternative suggestion for displaying the link inline. How about changing the `Fragment` in `ErrorNotice` to a `div` with `display: flex` and a suitable CSS `gap` value, and rendering the `Link` directly after the `ErrorText`? This would allow reusing `ErrorText` without adding complexity to it.

**Comment by nfmohit** [comment]
Good call, @techanvil. I actually was looking for a way to have it within the same `<p>` tag. I have updated the IB accordingly. Thank you!

**Comment by techanvil** [comment]
Thanks @nfmohit!

IB :white_check_mark:

**Comment by asvinb** [comment]
@sashadoes Technically the code looks good. However, UI wise, we have an issue on small screens:
![image](https://user-images.githubusercontent.com/31818/198996172-3cedc978-b1f9-4eb1-8b66-8c8a06e9c059.png)
As per the screenshot above, the layout is not ideal. We have a few options here:

1. Move the "Retry" button on its own line on small screens. However on medium and up screens, if we have a very long error text, we'll run into the same issue as on small screens.
![image](https://user-images.githubusercontent.com/31818/198996805-8f90595c-2da9-4b10-9eda-49c2cbb0437b.png)
2. Make the "Retry" button inline, where it's added seamlessly with the text.
![image](https://user-images.githubusercontent.com/31818/198996728-1fc96c28-0e46-47d2-b6d0-439a61f48ce0.png)

I think the second option is the best one but it'll require some changes to `assets/js/components/ErrorText.js` as well. Here is what I suggest:
* Add a new prop `children` to `assets/js/components/ErrorText.js` which will be displayed after the `p` tag if present.
* Update `assets/js/components/ErrorNotice.js` to render the "Retry" link as children of `ErrorText`.
* Update the styles so that the `p` within `.googlesitekit-error-text` is now an inline item, `display: inline`.

@nfmohit @techanvil Let me know what you think since you worked/reviewed the IB.
cc @felixarntz @aaemnnosttv

**Comment by nfmohit** [comment]
@asvinb I think the second option and your suggestions look perfect. Thank you!

**Comment by techanvil** [comment]
Thanks @asvinb & @nfmohit - I also agree with the proposed second option here. LGTM! Assigning back to @sashadoes for the followup. Cheers.

**Comment by mohitwp** [comment]
### QA Update ⚠️ 

@sashadoes @asvinb 

- Verified on dev.
- I'm able to recreate error on tag manager setup page and getting inline link for Retry. 

<details>

![image](https://user-images.githubusercontent.com/94359491/201036161-44f83b16-f09a-412f-9852-295db9281e8c.png)

![image](https://user-images.githubusercontent.com/94359491/201036199-e09d584a-83cf-48c3-9069-e3bb7d46020a.png)

![image](https://user-images.githubusercontent.com/94359491/201036233-7d6f7c5c-6878-47c6-aac3-b7082dba47fe.png)

</details>

**Question** - As per QAB I tried to recreate GA4 activation setup banner error notice.  I created error notice but I'm not getting inline "retry" link as mentioned under screenshot in QAB. Verified on both dev and latest release. On both I'm not getting "retry" option. Is it expected to have "retry" link for this error notice or it is just mock example ?

![image](https://user-images.githubusercontent.com/94359491/201038047-c4b7f4bb-81c0-4fa5-ad26-86f0e26614c9.png)

![image](https://user-images.githubusercontent.com/94359491/201039038-f3de9845-6c19-4577-834c-0e2ab732b7a2.png)

**Comment by sashadoes** [comment]
@asvinb it was a mock example. Sorry for confusing you.

**Comment by mohitwp** [comment]
### QA Update ✅ 

@sashadoes Thank you !

- Verified on dev.
- I'm able to recreate error on tag manager setup page and getting inline link for Retry using tweak.
- Retry button is working.
- Verified for both desktop and mobile view.

<details>

![image](https://user-images.githubusercontent.com/94359491/201601840-20a73056-9c07-444f-b6c5-2228ca9b709e.png)

![image](https://user-images.githubusercontent.com/94359491/201601875-c3b627eb-2189-42bf-8219-ff9878d27bd1.png)

![image](https://user-images.githubusercontent.com/94359491/201601936-8bd595f6-7518-4218-a43d-9e2d80c1bd06.png)

</details>

**Comment by felixarntz** [comment]
## Approval ❌ 

@techanvil @nfmohit @sashadoes After taking a closer look, this issue's implementation will unfortunately need to be reverted. This is not due to the implementation itself, but rather because the ACs here are not really helping with the UX problem from before, they just introduce another new UX problem.

After reviewing this with @aaemnnosttv and @marrrmarrr, while both versions of the UI are problematic, it's better for now to stick to the old one. That said, we should definitely address this in a proper way, for which I've opened #6162.

The PR for this issue should be reverted and afterwards it can be closed as `invalid`.

**Comment by techanvil** [comment]
Thanks @felixarntz, I have created a PR to revert the change: #6165

**Comment by aaemnnosttv** [comment]
This has been reverted, and removed from the changelog. Closing in favor of #6162 👍

**Comment by wpdarren** [comment]
### QA Update: ✅ 

- Ran through a quick regression test to ensure that the code was reverted and didn't cause any issues.

https://user-images.githubusercontent.com/73545194/202904023-d303a643-bd29-4fa3-8265-07de90f2a0f3.mp4
